### DISCUSSION TITLE
ctrl + w  behaviour: jump in front of separating characters?

### DISCUSSION THREAD

**Discussion by T35R6braPwgDJKq** [ohmyzsh/ohmyzsh#10137] [Help] [answered]
my `$WORDCHARS` is set to `''`.
Consider the following path: `ls -l -a --color /long/path/with-some-things/to/consider.txt`
the following lines represent the modification by pressing <kbd>ctrl</kbd> + <kbd>w</kbd> each time:
```
ls -l -a --color /long/path/with-some-things/to/consider.
ls -l -a --color /long/path/with-some-things/to/
ls -l -a --color /long/path/with-some-things/
ls -l -a --color /long/path/with-some-
ls -l -a --color /long/path/with-
ls -l -a --color /long/path/
ls -l -a --color /long/
ls -l -a --color /
ls -l -a --
```
What I actually want is that it goes from 
```ls -l -a --color /```
to 
```ls -l -a --color```
and after that to 
```ls -l -a```

is that even possible just with setting the `WORDCHARS` accordingly?
It seems the problem would be solved by jumping before the separator and not staying behind it
That would result in the following sequence (I guess)
```
ls -l -a --color /long/path/with-some-things/to/consider
ls -l -a --color /long/path/with-some-things/to
ls -l -a --color /long/path/with-some-things
ls -l -a --color /long/path/with-some
ls -l -a --color /long/path/with
ls -l -a --color /long/path
ls -l -a --color /long
ls -l -a --color # mind the blank, which is unwanted too
ls -l -a # that unwanted blank again
```

**Comment by mcornella** [comment, marked as answer]
This question has been answered in the Zsh mailing list: https://zsh.org/mla/users/2021/msg00686.html
